app/main.py

```diff
@@ -1,8 +1,41 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: str, clean_mark: str,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: str,
+                 clean_power: int, average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if car.clean_mark < self.clean_power:
+            cost = (
+                car.comfort_class * (self.clean_power - car.clean_mark) * self.average_rating
```

Comment:
The expression `car.comfort_class * (self.clean_power - car.clean_mark) * self.average_rating` is problematic because `comfort_class` is a string, and `clean_mark` is also a string. Ensure that these values are converted to appropriate numeric types before performing arithmetic operations.